compiler/qsc_doc_gen/src/generate_docs.rs

```diff
@@ -22,6 +23,136 @@ use std::sync::Arc;
 type Files = Vec<(Arc<str>, Arc<str>, Arc<str>)>;
 type FilesWithMetadata = Vec<(Arc<str>, Arc<Metadata>, Arc<str>)>;
 
+// Namespace -> metadata for items
+type ToC = FxHashMap<Rc<str>, Vec<Arc<Metadata>>>;
```

Comment:
Why this mix of `Rc` and `Arc` in this file? Do you need `Arc` at all, since this is all single threaded?

Reply:
Strange, I definitely needed them at one point, because I remember having to go out of my way to use them, but I can't remember why. Turning them back to Rc's works fine now, so we'll go with that. I've turned them back into Rc's.